utils.js

```diff
@@ -183,25 +183,60 @@ CTLEventUtils.readURLParams = function(eventsList, queryString, index) {
             case 'q':
                 filteredEvents = CTLEventUtils.searchEvents(
                     filteredEvents, index, splitParam[1]);
+                CTLEventUtils.populateQ(splitParam[1]);
```

Comment:
It seems like `readURLParams` is now in charge of two things: reading the URL params and populating the inputs. Another way to organize this could be to use the output of `readURLParams` to populate the inputs, using a `populateInputs()` function.